integrations/spring/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.apache.opendal</groupId>
+    <artifactId>spring-opendal-parent</artifactId>
```

Comment:
I take a look over `spring-boot-starter-data-redis` and found that they are in three `artifactId`:

- `spring-data-redis`
- `spring-boot-starter-data-redis`
- `spring-boot-starter-data-redis-reactive`

Should we follow the same pattern that have three `artifact` instead of `module`?

Reply:
@Xuanwo I think you may misunderstand module and artifactId's definition. Every library has an artifactId, which is the smallest unit of reference. Some libraries are responsible for managing other libraries and do not directly contain code themselves; they use a <module> to manage modules or sub-projects (in Maven 4.0, these will be referred to as sub-projects instead of modules).